app/assets/javascripts/plate-map/plate-layout.js

```diff
@@ -167,5 +173,112 @@ $.widget("DNA.plateLayOut", {
   // wellsData follows syntax: {0:{field1: val1, field2: val2}, 1:{field1: val1, field2: val2}}
   getWellsDifferences: function(wellsData) {
     return this.getDifferentWellsVals(wellsData);
+  },
+
+  disableAllFields: function(){
+    var numericFields = $('.plate-setup-tab-input, .plate-setup-tab-label-select-field, ' +
+      '.plate-setup-tab-multiselect-field, .plate-setup-tab-select-field');
+    for (var i = 0; i < numericFields.length; i++) {
+      numericFields[i].disabled = true;
+    }
+    $(".plate-setup-tab-name-singleSelect").text("Select to inspect");
+  },
+
+  enableAllFields: function(){
+    var numericFields = $('.plate-setup-tab-input, .plate-setup-tab-label-select-field, ' +
+      '.plate-setup-tab-multiselect-field, .plate-setup-tab-select-field');
+    for (var i = 0; i < numericFields.length; i++) {
+      numericFields[i].disabled = false;
+    }
+    $(".plate-setup-tab-name-singleSelect").text("Select to Edit")
+  },
+
+  readOnly: null,
+  isReadOnly: function(flag){
+    if (flag){
+      this.readOnly = true;
+      this.readOnlyHandler = function(){
+        $("#plate-map-control-button-container-id").css("display", "none");
+        this.disableAllFields();
+        $('.multiple-field-manage-delete-button').css("display", "none");
+      };
+    } else {
+      this.readOnly = false;
+      this.readOnlyHandler = function(){
+        $("#plate-map-control-button-container-id").css("display", "flex");
```

Comment:
Use `that.overLayButtonContainer`, not id

Reply:
fixed